src/Core/Components/Progress/FluentProgressRing.razor.cs

```diff
@@ -44,4 +51,38 @@ public partial class FluentProgressRing : FluentComponentBase
     /// </summary>
     [Parameter]
     public RenderFragment? ChildContent { get; set; }
+
+    /// <summary>
+    /// Gets or sets the component width and height.
+    /// </summary>
+    [Parameter]
+    public string? Width { get; set; }
+
+    /// <summary>
+    /// Gets or sets the stroke width of the progress ring. If not set, the default theme stroke width is used.
+    /// </summary>
+    [Parameter]
+    public ProgressStrokes Stroke { get; set; } = ProgressStrokes.Normal;
+
+    /// <summary>
+    /// Gets or sets the color to be used for the progress ring. If not set, the default theme color is used.
+    /// </summary>
+    [Parameter]
+    public string? Color { get; set; }
+
+    private (int Width, int Radius, int Dashoffset) StrokeDetails => Stroke switch
+    {
+        ProgressStrokes.Thin => (1, 7, 0),
+        ProgressStrokes.Normal => (2, 7, 0),
+        ProgressStrokes.Large => (4, 6, (int)(0.066 * (Value ?? 0) + 0.22)),
```

Comment:
Thin and Large aren't the opposites of each other. Enum values should either be Thin/Normal/Thick or Small/Normal/Large.

Should ProgressStrokes be ProgressStroke? Enum names aren't pluralized. e.g. StringComparison, not StringComparisons.

Reply:
Good comments. Thank you.

I'm going to use Small/Normal/Large: easier to extend to ExtraLarge and others if necessary.

I agree with the Enum name.

Reply:
Fixed in #1125